content/blog/2024-02-26-mainnet-hardfork.mdx

```diff
@@ -0,0 +1,60 @@
+---
+author: elena
+title: Mainnet Activation for Hardfork-1
+description: Everything you need to know for the upcoming hardfork-1 on Iron Fish mainnet 
+image: /images/blog/mainnet_hardfork.png
+tags: [ironfish, iron fish, hardfork, hard-fork, hard fork, activation, mainnet]
+---
+
+# Mainnet Activation for Hardfork-1
+
+Iron Fish is preparing for its first hardfork. The first hardfork will include:
+- [FIP-2](https://fips.ironfish.network/fips/fip-2-enforce-sequential-block-time) : Enforce sequential block time (a block’s timestamp has to be greater than the timestamp of its parent) 
+- [FIP-3](https://fips.ironfish.network/fips/fip-3-memory-hard-mining-algorithm) : Switch hashing algorithm from blake3 to FishHash
+- [FIP-8](https://fips.ironfish.network/fips/fip-8-difficulty-adjustment) : Difficulty adjustment (to accommodate the algorithm change)
+- [FIP-9](https://fips.ironfish.network/fips/fip-9-swap-randomness-and-graffiti) : Swap randomness and graffiti field positions in the Block Header
+
+The most notable change is the hashing algorithm switch from blake3 to FishHash. To read more about FishHash, the motivation behind finding a new hashing algorithm for Iron Fish, as well as exploration of other algorithm choices, please see [FIP-3](https://fips.ironfish.network/fips/fip-3-memory-hard-mining-algorithm) (or community created video walkthroughs like this [one](https://youtu.be/1R7VffTzM9g?si=U0PhustQsp8TG9ak)). Please note that [FIP-9](https://fips.ironfish.network/fips/fip-9-swap-randomness-and-graffiti) that is included in this hardfork also changes the Block Header format for Iron Fish. 
```

Comment:
Don't think we need to call out FIP-9 again explicitly here as it only changes the block header when mining. Not changing the block header in other contexts like networking layer or block explorer